src/btree/bt_split.c

```diff
@@ -586,6 +586,16 @@ __split_deepen(WT_SESSION_IMPL *session, WT_PAGE *parent, uint32_t children)
 	 * fails, we don't roll back that change, because threads may already
 	 * be using the new index.
 	 */
+	for (parent_refp = pindex->index, i = 0;
```

Comment:
I understand that we are inside a split so it's probably safe, but I'd be inclined to use ```WT_INTL_FOREACH_BEGIN``` style loop here. Is there a reason not to?